packages/inventory-insights/src/Insights.js

```diff
@@ -92,46 +109,79 @@ class InventoryRuleList extends Component {
-                                        }
-                                    </SplitItem>
-                                </Split>
-                                <Section type='icon-group__with-major' className='batterySectionOverrides smallFontSizeOverride'>
-                                    <Battery label='Impact' severity={ rule.impact.impact }/>
-                                    <Battery label='Likelihood' severity={ rule.likelihood }/>
-                                    <Battery label='Total Risk' severity={ rule.total_risk }/>
-                                    <Battery label='Risk Of Change' severity={ value.resolution.resolution_risk.risk }/>
-                                </Section>
-                            </Fragment>
+                            title: <div key={ key }>
+                                { moment(rule.publish_date).fromNow() }
+                            </div>
+                        },
+                        {
+                            title: <div className='pf-m-center' key={ key } style={ { verticalAlign: 'top' } }>
+                                <Battery
+                                    label='Total risk'
+                                    labelHidden
+                                    severity={ rule.total_risk }
+                                />
+                            </div>
+                        }, {
+                            title: <div className='pf-m-center' key={ key } style={ { verticalAlign: 'top' } }>
+                                <Battery
+                                    label='Risk of change'
+                                    labelHidden
+                                    severity={ value.resolution.resolution_risk.risk }
+                                />
+                            </div>
+                        },
+                        {
+                            title: <div className='pf-m-center ' key={ key }>
+                                { value.resolution.has_playbook &&
+                                <CheckIcon className='successColorOverride'/> }
+                            </div>
                         }
                     ]
                 },
                 {
-                    parent,
+                    parent: key,
                     fullWidth: true,
                     cells: [{
                         title: <ReportDetails key={ `child-${key}` } report={ value } kbaDetail={ kbaDetail }/>
                     }]
                 }
             ];
+
+            const isValidSearchValue = searchValue.length === 0 || rule.description.search(new RegExp(searchValue, 'i')) !== -1;
+            const isValidFilterValue = Object.keys(filters).length === 0 || Object.keys(filters).map((key) => {
+                const filterValues = filters[key];
+                let rowValue;
+                switch (key) {
```

Comment:
I am not that big of a fan of switch, what do you think about this
```JS
function rowValue(value, key) {
  return ({
  'has_playbook': value.resolution[key],
  risk: value.resolution.resolution_risk[key]
}) || value[key]
}
```

And getting the value would be `rowValue(value, key)`

Reply:
OH yeah i can do that! <3 <3 <3

Reply:
eehhhh doesn't quite work like that, would a pile of ifs be preferred to the switch?

Reply:
ok lets just make em objects, thats a lil simpler